src/utilities/manifest-stability-utility.ts

```diff
@@ -59,15 +70,19 @@ export async function checkPodStatus(kubectl: Kubectl, podName: string): Promise
         case 'Running':
             if (isPodReady(podStatus)) {
                 console.log(`pod/${podName} is successfully rolled out`);
+            } else {
+                kubectl.describe('pod', podName);
             }
             break;
         case 'Pending':
             if (!isPodReady(podStatus)) {
                 core.warning(`pod/${podName} rollout status check timedout`);
+                kubectl.describe('pod', podName);
             }
             break;
         case 'Failed':
             core.error(`pod/${podName} rollout failed`);
+            kubectl.describe('pod', podName);

```

Comment:
The call during errors seems to be spread out. Try to call it at one point, will come in handy if/when you want to add some extra stuff later

Reply:
Should I create a function and put describe command in it?

Reply:
Set a variable for indicator and do the describe call in one place